### PR TITLE
Fix Arkansas gross income exemptions

### LINKED ISSUE #4119: Fix Arkansas gross income exemptions
In the 0.688.1 code, social security benefits are excluded from AR AGI even though social security benefits are not included in AR AGI.   This causes AR AGI amounts to be too small in Policyengine-US.

Also, the AR income tax form and instructions seem to imply that, in order to calculate AR AGI correctly in all cases, Policyengine-US needs to replace the `ar_agi` variable with new `ar_agi_indiv` and `ar_agi_joint` variables.

### PR DESCRIPTION
**Pull request by PavelMakarchuk**
Fixes #4119

### CODE REVIEW

File: policyengine_us/parameters/gov/states/ar/tax/income/exemptions/exemptions.yaml [hunk 1]
```diff
@@ -1,16 +1,6 @@
 description: Arkansas accounts for these exemptions when calculating the state adjusted gross income.
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/ar/tax/income/exemptions/exemptions.yaml:1]: Is this accurate? Is it AGI or gross income?
```suggestion
description: Arkansas exempts these sources from state adjusted gross income.
```

**Review by MaxGhenis — APPROVED** [review]
good to merge after clarifying this

File: policyengine_us/parameters/gov/states/ar/tax/income/exemptions/exemptions.yaml (hunk 1 above)
Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/ar/tax/income/exemptions/exemptions.yaml:1]: should be gross income

File: policyengine_us/parameters/gov/states/ar/tax/income/exemptions/exemptions.yaml [hunk 2]
```diff
@@ -1,21 +1,12 @@
-description: Arkansas accounts for these exemptions when calculating the state adjusted gross income.
+description: Arkansas exempts these sources from state adjusted gross income.
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/ar/tax/income/exemptions/exemptions.yaml:1]: Please also update the issue and PR title
```suggestion
description: Arkansas exempts these sources from state gross income.
```

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**